README.md

```diff
@@ -179,31 +179,30 @@ Add to your `.babelrc`'s plugins array.
 
 Benchmark Results for react.js:
 
-Input Size: 133.2kB
+Input Size: 141.63kB
 
-Input Size (gzip): 31.03kB
+Input Size (gzip): 33.66kB
 
-| minifier              | output raw | raw win | gzip output | gzip win | parse time (ms) | minify time (ms) |
-| --------------------- | ---------- | ------- | ----------- | -------- | --------------- | ---------------- |
-| **babili**            | 37.89kB    | 72%     | 12.54kB     | 60%      | 2.39            | 2262.64          |
-| **uglify**            | **37.7kB** | **72%** | **12.4kB**  | **60%**  | **2.68**        | **1019.53**      |
-| **closureCompiler**   | 36.73kB    | 72%     | 12.45kB     | 60%      | 3.01            | 2692.87          |
-| **closureCompilerJs** | 44.58kB    | 67%     | 14.54kB     | 53%      | 2.89            | 6591.58          |
-| **butternut**         | 39.2kB     | 71%     | 12.81kB     | 59%      | 2.88            | 281.72           |
+| minifier              | output raw  | raw win | gzip output | gzip win | parse time (ms) | minify time (ms) |
+| --------------------- | ----------- | ------- | ----------- | -------- | --------------- | ---------------- |
+| **babelMinify**       | 40.63kB     | 71%     | 13.33kB     | 60%      | 2.40            | 1879.97          |
```

Comment:
shouldnt it be listed as `babel-minify`? Would look better and correspond to the actual name of the package